simulate/simulate_test.go

```diff
@@ -24,7 +24,7 @@ func TestRandGene(t *testing.T) {
 		if len(a[0].Seq) != test.length {
 			t.Errorf("expected RandGene to give %v, gave %v", test.length, len(a[0].Seq))
 		}
-		//fmt.Print(dna.BasesToString(a[0].Seq), "\n")
+		fmt.Print(dna.BasesToString(a[0].Seq), "\n")
```

Comment:
Can we leave this commented out?  When we scroll the screen during tests, I think it is harder to notice what fails.  That way we have a test, when it fails someone can go into the file and uncomment this to try and debug it.